src/runtime/scope.rs

```diff
@@ -212,6 +218,50 @@ impl RuntimeScope {
         (handle, cx, abort_reg)
     }
 
+    /// Spawns a new, plain task.
+    pub async fn spawn_task<T, Sup>(&mut self, mut task: T, supervisor_addr: Addr<Sup>) -> AbortHandle
+    where
+        T: Task + 'static,
+        Sup: 'static + HandleEvent<TaskReport<T>>,
+    {
+        let (abort_handle, abort_registration) = AbortHandle::new_pair();
+        let mut child_scope = self.child(None, Some(abort_handle.clone())).await;
+        let child_task = spawn_task(task.name().as_ref(), async move {
+            let fut = task.run();
+            let res = Abortable::new(AssertUnwindSafe(fut).catch_unwind(), abort_registration).await;
+            child_scope.abort().await;
```

Comment:
We probably shouldn't abort the children on exit, similar to how I remove this from the actor, as this allows the task to spawn children if it wants to and manage their lifecycles better.